wiki/Tournaments/TWC/3/en.md

```diff
@@ -1,13 +1,13 @@
 ---
 tags:
-- TWC #3
+- 'TWC #3'
```

Comment:
are those quotes needed?

Reply:
Isn't it to escape? Or not neccesary?

Reply:
idk i have never seen quotes in tags before but i think they are not needed

Reply:
they are needed, this is YAML and `#` (with whitespace before it) marks a comment